packages/core/src/submodules/httpAuthSchemes/aws_sdk/NODE_AUTH_SCHEME_PREFERENCE_OPTIONS.ts

```diff
@@ -14,7 +17,11 @@ export const NODE_AUTH_SCHEME_PREFERENCE_OPTIONS: LoadedConfigSelectors<string[]
    * @param env - Node process environment object
    * @returns Array of auth scheme strings if preference is set, undefined otherwise
    */
-  environmentVariableSelector: (env: NodeJS.ProcessEnv) => {
+  environmentVariableSelector: (env: NodeJS.ProcessEnv, options?: EnvOptions) => {
+    if (options?.signingName) {
+      const bearerTokenKey = getBearerTokenEnvKey(options.signingName);
+      if (bearerTokenKey in env) return ["httpBearerAuth"];
```

Comment:
should this only activate if `!(NODE_AUTH_SCHEME_PREFERENCE_ENV_KEY in env)` is true?

Reply:
No. Confirmed internally that token is more specific